### ISSUE TITLE
Jekyll slow to compile site?

### ISSUE THREAD

**Issue by sdellis** [code4lib/2016.code4lib.org#105]
Is anyone else having this problem?  It makes it hard to test people's changes when I have to wait nearly three minutes for it to generate...

```
Regenerating: 2 file(s) changed at 2015-09-17 18:20:07 ...done in 178.773342 seconds.
```

**Comment by queryluke** [comment]
Just tried some random changes, finished in .9, 1.3, and 0.3 sec

**Comment by cdmo** [comment]
Yeah, same here, I'm seeing `0.565891 seconds` and the like... maybe your system is maxed out?

**Comment by sdellis** [comment]
Ok, well, I'm actually getting a new laptop today and I will see if there's a difference.  However, I did notice that by removing the "prototype" directory, it finishes in around .5 secs.  This is a lot of the Bootstrap less stuff and the prototype pages, which at this point, we're not really using and could be removed.  However, I tried moving the bootstrap.css that it generates into /css and pointing to it, but I lost styles that are generated via less (variable colors, etc.).  Anyway, at least there's progress on this and I'll try to resolve it soon.
